package/src/components/ImageGallery/components/ImageGalleryHeader.tsx

```diff
@@ -29,7 +29,7 @@ const styles = StyleSheet.create({
   },
   innerContainer: {
     flexDirection: 'row',
-    height: 56,
+    minHeight: 56,
```

Comment:
Just to give some better context around this, this would make sure the entire header doesn't break if font scaling is enabled. Although we don't yet support accessibility features completely within the SDK, this was an easy win we could do to help.